source/Heat/IntroducingOpenStackHeat.rst

```diff
@@ -1,246 +1,246 @@
+
+    **Note:** While Heat can interpret CFN (CloudFormation) Templates, they are _not_ backwards compatible with Heat Orchestrated Templates. It is recommended to use Heat Orchestrated Templates to create stacks.
+
+
+Heat Orchestrated Templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Heat Orchestrated Templates are YAML files that instruct Heat which resources to create and the relationships between resources. Ansible has documentation on how to write YAML files that can be found here: https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html
+
+Heat Templates consist of seven sections:
+
+1. ``heat_template_version:`` (**required**) Indicates which format and features are used and supported when creating the stack.
+
+2. ``description:`` (optional) Description of the stack template. It is recommended to include a description in templates to describe what users can do with the template.
+
+3. ``parameter_groups:`` (optional) Defines how to group input parameters and the order of the parameters.
+
+4. ``parameters:`` (optional) Defines input parameters. This section can be omitted if there are no input values required.
+
+5. ``resources:`` (**required**) Defines resources in the template. At least one resource should be defined in a HOT template.
+
+6. ``outputs:`` (optional) Defines output parameters available to users once the template has been instantiated.
+
+7. ``conditions:`` (optional) Includes statements which can be used to apply conditions to a resource, for example a resource is created only when a property is defined or when another resource has been created first.
+
+
+The structure of a HOT template is given as:
+
+.. code-block:: yaml
+
+  heat_template_version: 2018-08-31 #OpenStack Version we want to use.
+                                    #Here, we want to use template for the Rocky release onwards
+
+  description: #description of the template
+
+  parameter_groups: #declares the parameter group and order.
+    #This is not a compulsory section, however it is useful for grouping
+    #parameters together when building more complex templates.
+
+  parameters: #declares the parameters for resources
+
+  resources: #declares the template resources
+  # e.g. alarms, floating IPs, instances etc.
+
+  outputs: #declares the output of the stack
+
+  conditions: #declares any conditions on the stack
+
+
+Please see the documentation **Create a Heat Stack** (https://stfc-cloud-docs.readthedocs.io/en/latest/howto/CreateAHeatStack.html) for an introduction to creating a stack using a HOT template.
+
+
+Rocky Heat Templates
+^^^^^^^^^^^^^^^^^^^^^
+
+Templates which use:
+
+.. code-block:: yaml
+
+  heat_template_version: 2018-08-31
+
+  #Or
+
+  heat_template_version: rocky
+
+Indicate that the template is a HOT template and has features added and/or removed up to the Train Release. The list of supported functions in a Rocky Heat Template is:
```

Comment:
This is technically up to Rocky release, but Train uses Rocky Heat templates so we can keep this description.